modules/ROOT/pages/connected-app-bearer-token-example.adoc

```diff
@@ -0,0 +1,64 @@
+= Getting the Bearer Token for a Connected App Example
+
+This example demonstrates how to create a connected app and get the bearer token so that you can invoke Anypoint Platform APIs. 
+
+[[create-connected-app]]
+== Create a Connected App
+
+include::partial$include-nav-connected-apps.adoc[]
+
+. In the *Owned Apps* section, click *Create App*.
+. In the *Create App* page, enter a unique *Name* for the app, select *App acts on its own behalf (client credentials)*, and then click *Save*.
+
+[[add-required-scopes]]
+== Add the Required Scopes
+
+To deploy the app to CloudHub, you must add Runtime Manager scopes to the app.
+
+. On the *Connected Apps* page, click the name of the app to add permissions to. 
+. In the *Update App* window, click *Add Scopes*.
```

Comment:
We don't have to do Create App and Add Scopes as two different steps. On the 'Create App' page itself, once the name is entered and the type of app is selected, we can click on the `Add Scopes` button to open the modal that lets us select the required scopes before hitting save.